source/download/ovirt-live.html.md

```diff
@@ -7,6 +7,10 @@ authors: apevec, bproffitt, dneary, jvandewege, mgoldboi, obasan, sandrobonazzol
 
 # oVirt Live
 
+**Note: oVirt Live is now considered deprecated and is no longer be maintained.**
+
+Please consider the use of [oVirt Node](../../node/) instead.
```

Comment:
Thanks for the patch, the deprecation notice is indeed useful. About the suggestion on using Node instead, Node is not really covering the demo functionalities provided by oVirt Live. We are working on a demo tool to replace oVirt Live